Linker errors with afl++ due to wrong CC/CXX, need to use env vars

Hi,

different projects ran into linker errors with afl today, see
https://oss-fuzz-build-logs.storage.googleapis.com/log-d9c44c8c-807c-4b40-b5bc-3e7d037ff36e.txt
https://oss-fuzz-build-logs.storage.googleapis.com/log-f199b794-0f4d-4de9-b308-c4536ad68cc9.txt
```
Step #22: /usr/lib/libFuzzingEngine.a(aflpp_driver.o): In function `main':
Step #22: /src/aflplusplus/utils/aflpp_driver/aflpp_driver.c:236: undefined reference to `__afl_manual_init'
Step #22: /src/aflplusplus/utils/aflpp_driver/aflpp_driver.c:244: undefined reference to `__afl_manual_init'
Step #22: /src/aflplusplus/utils/aflpp_driver/aflpp_driver.c:251: undefined reference to `__afl_persistent_loop'
Step #22: /src/aflplusplus/utils/aflpp_driver/aflpp_driver.c:251: undefined reference to `__afl_persistent_loop'
Step #22: /src/aflplusplus/utils/aflpp_driver/aflpp_driver.c:262: undefined reference to `__afl_fuzz_len'
Step #22: /src/aflplusplus/utils/aflpp_driver/aflpp_driver.c:265: undefined reference to `__afl_fuzz_ptr'
```
Did we make the same mistake in our projects or is there something wrong with your afl setup? I'd appreciate if you could have a look.

Cheers,
Robert

--- a/projects/ibmswtpm2/no_writes.patch
+++ b/projects/ibmswtpm2/no_writes.patch
@@ -88,8 +88,8 @@ index cc3e410..c10ba5a 100644
  
  
  CC = /usr/bin/gcc
-+CC = clang
-+CXX = clang++
++CC = ${CC}
++CXX = ${CXX}
  
 -CCFLAGS = -Wall  			\
 +CCFLAGS = $(CFLAGS) -Wall  			\

--- a/projects/ibmswtpm2/no_writes.patch
+++ b/projects/ibmswtpm2/no_writes.patch
@@ -84,12 +84,10 @@ diff --git a/makefile b/makefile
 index cc3e410..c10ba5a 100644
 --- a/makefile
 +++ b/makefile
-@@ -40,16 +40,19 @@
+@@ -40,16 +40,16 @@
  
  
- CC = /usr/bin/gcc
-+CC = ${CC}
-+CXX = ${CXX}
+-CC = /usr/bin/gcc
  
 -CCFLAGS = -Wall  			\
 +CCFLAGS = $(CFLAGS) -Wall  			\

